src/Admin/AbstractAdmin.php

```diff
@@ -1487,6 +1489,18 @@ public function attachAdminClass(FieldDescriptionInterface $fieldDescription)
 
             if (!$pool->hasAdminByClass($targetModel)) {
                 return;
+                // NEXT_MAJOR: Uncomment the following exception instead.
+//                throw new \InvalidArgumentException(sprintf(
+//                    'No admin found for the class "%s", please use the admin_code option instead',
+//                    $targetModel
+//                ));
+            }
+
+            if (!$pool->hasSingleAdminByClass($targetModel)) {
+                throw new \InvalidArgumentException(sprintf(
+                    'To many admin found for the class "%s", please use the admin_code option instead',
```

Comment:
You check on not single but says that too many( If someone delete method above here message logic will be broken.

Reply:
I don't understand, because the method above is not suppose to be deleted.

Reply:
Ok